src/components/NewWorkspaceModal.js

```diff
@@ -246,9 +246,27 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
           id,
           value: bucketLocation,
           onChange: ({ value }) => setBucketLocation(value),
-          options: _.sortBy('label', isAlphaRegionalityUser ? allRegions : availableBucketRegions)
+          options: isAlphaRegionalityUser ? allRegions : availableBucketRegions
         })
       ])]),
+      isLocationMultiRegion(bucketLocation) && div({ style: { ...warningStyle } }, [
+        icon('warning-standard', { size: 24, style: { color: colors.warning(), flex: 'none', marginRight: '0.5rem' } }),
+        div({ style: { flex: 1 } }, [
+          `Effective October 1, 2022, Google Cloud will charge egress fees on data stored in multi-region storage buckets.`,
+          p(`Choosing a multi-region bucket location may result in additional storage costs for your workspace.`),
+          p(
+            [
+              `Unless you require geo-redundancy for maximum availabity for your data, you should choose a single region bucket location.`,
+              h(Link, { href: 'https://terra.bio/moving-away-from-multi-regional-storage-buckets', ...Utils.newTabLinkProps },
```

Comment:
This link isn't working for me-- I expect it just hasn't been made public yet?

Reply:
Correct, will double check re: timing w/adam.